app/controllers/reservations_controller.rb

```diff
@@ -0,0 +1,15 @@
+class ReservationsController < ApplicationController
+  def new
+  end
+
+  def create
+  end
+
+  #def index
```

Comment:
Incorrect indentation detected (column 2 instead of 4).<br>Missing space after #.